about.html

```diff
@@ -0,0 +1,111 @@
+    </nav>
+    <section class="section-one padding-x">
+        <h3>"Hallo sharing World"</h3>
+        <h1>CREATIVE COMMONS <br> GLOBAL SUBMMIT 2015</h1>
+        <div class="ipsum">
+            <p>It has survived not only five 
+                centuries, but also the leap into electronic typesetting, remaining essentially unchanged. </p><p>
+                It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum 
+                passages, and more recently with desktop publishing software like Aldus PageMaker including
+                versions of Lorem Ipsum.</p>
+        </div>
+        <div>
+            <p class="p">Please contact per email for any further queston about about CC Global summit 2015! </p>
+            <p class="email">ccsummit2015@cckorea.org</p>
+        </div>
+    </section>
+    <section class="section-two padding-x">
+        <h1>Creative common Global summit 2015 logo</h1>
+        <span class="red_underline"></span>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard 
+            dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a 
+            type specimen book. It has survived not only five centuries, but also the leap into electronic 
+            typesetting, remaining essentially unchanged.
+        </p>
+        <div>
+            <img class="logo" src="./imgs/icons/logo.png" alt="Logo Images">
+        </div>
+    </section>
+    <section class="section-three padding-x">
+        <h1>See the pass Global cc summits</h1>
+        <span class="red_underline"></span>
+        <p>
+            Tke a look at past two cc global summit which took place in benousalians and in warsaws
+        </p>
+        <div>
+            <section class="image-1"><p><b>2015</b>Event on last year while students was celebrationg their day.</p></section>
+            <section class="image-2"><p><b>2018</b> on last year while students was celebrationg their day.</p></section>
+        </div>
+    </section>
+    <section class="parnter-section-about padding-x">
+        <h2>Partner</h2>
+        <span class="red_underline"></span>
+        <div class="partner-content">
+            <div>
+                <img src="./imgs/background/dtu.png" alt="DTU">
+                <p>DTU</p>
+            </div>
+            <div>
+                <img src="./imgs/background/uog.png" alt="UoG">
+                <p>UoG</p>
+            </div>
+            <div>
+                <img src="./imgs/background/Arba_Minch_University.png" alt="AMU">
+                <p>AMU</p>
+            </div>
+            <div>
+                <img src="./imgs/background/MoE.png" alt="MoE">
+                <p>MoE</p>
+            </div>
+            <div>
+                <img src="./imgs/background/dtu.png" alt="DTU">
+                <p>BDU</p>
+            </div>
+        </div>
+    </section>
```

Comment:
as suggested previously Kindly update the partner section in your about page to match the design better. You can try reducing the size of the images
![image](https://user-images.githubusercontent.com/60612329/234568489-e79288da-949a-4e7d-8cfa-19e3192f107a.png)

Reply:
## ✋✋ On my device the layout of the partner section on the About page looks as follows, which is similar to the original guideline.
👇 👇 Layout on my device 👇 👇 
![image](https://user-images.githubusercontent.com/16529009/234584166-abb9ece1-51e4-42b9-ad4b-f6412ff1d167.png)